### ISSUE TITLE
since a few Weeks, IGN high quality format gets an 403 Error. 

### ISSUE THREAD

**Issue by rubbishgee** [ytdl-org/youtube-dl#31108]
If you try it with a video that is about one month old, everything works flawless. Is this an issue that can be fixed? 

```
% yt-dlp https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022 --verbose
[debug] Command-line config: ['https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.06.29 [9d339c4]
[debug] Python 3.10.5 (CPython 64bit) - macOS-11.6-x86_64-i386-64bit 
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 4.3, ffprobe 4.3, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [ign.com] Extracting URL: https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022
[ign.com] john-wick-4-teaser-trailer-comic-con-2022: Downloading JSON metadata
[ign.com] 512a96f4-9e32-42b6-9b37-43b1e3e7eb1d: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 512a96f4-9e32-42b6-9b37-43b1e3e7eb1d: Downloading 1 format(s): mezzanine
[debug] Invoking http downloader on "https://ign-video-prd-assets.s3.amazonaws.com/2022/07/23/jw4-sneak-peek-1440p-24fps-vp9-160kbit-opus-1658545184410.webm"
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/Cellar/yt-dlp/2022.6.29/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 3129, in process_info
    success, real_download = self.dl(temp_filename, info_dict)
  File "/usr/local/Cellar/yt-dlp/2022.6.29/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 2829, in dl
    return fd.download(name, new_info, subtitle)
  File "/usr/local/Cellar/yt-dlp/2022.6.29/libexec/lib/python3.10/site-packages/yt_dlp/downloader/common.py", line 444, in download
    ret = self.real_download(filename, info_dict)
  File "/usr/local/Cellar/yt-dlp/2022.6.29/libexec/lib/python3.10/site-packages/yt_dlp/downloader/http.py", line 372, in real_download
    establish_connection()
  File "/usr/local/Cellar/yt-dlp/2022.6.29/libexec/lib/python3.10/site-packages/yt_dlp/downloader/http.py", line 131, in establish_connection
    ctx.data = self.ydl.urlopen(request)
  File "/usr/local/Cellar/yt-dlp/2022.6.29/libexec/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 3596, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/local/Cellar/python@3.10/3.10.5/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/local/Cellar/python@3.10/3.10.5/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/local/Cellar/python@3.10/3.10.5/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/local/Cellar/python@3.10/3.10.5/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python@3.10/3.10.5/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

**Comment by 89z** [comment]
This isn't YT-DLP repo.

**Comment by dirkf** [comment]
yt-dl and yt-dlp show the same behaviour.
```shellsession
$ python -m youtube_dl -v -F 'https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-F', u'https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: 1ff18694c
[debug] Python version 2.7.17 (CPython) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[ign.com] john-wick-4-teaser-trailer-comic-con-2022: Downloading JSON metadata
[ign.com] 512a96f4-9e32-42b6-9b37-43b1e3e7eb1d: Downloading m3u8 information
[info] Available formats for 512a96f4-9e32-42b6-9b37-43b1e3e7eb1d:
format code  extension  resolution note
0            mp4        640x360     344k , 23fps
hls-447      mp4        640x360     447k , avc1.4d401e, 23.976fps, mp4a.40.2
2            mp4        852x480     718k , 23fps
hls-873      mp4        852x480     873k , avc1.4d401e, 23.976fps, mp4a.40.2
4            mp4        960x540    1120k , 23fps
hls-1278     mp4        960x540    1278k , avc1.4d401f, 23.976fps, mp4a.40.2
6            mp4        1280x720   1891k , 23fps
hls-2094     mp4        1280x720   2094k , avc1.64001f, 23.976fps, mp4a.40.2
8            mp4        1920x1080  3363k , 23fps
hls-3660     mp4        1920x1080  3660k , avc1.640028, 23.976fps, mp4a.40.2
10           mp4        2560x1440  6786k , 23fps
hls-7185     mp4        2560x1440  7185k , avc1.640032, 23.976fps, mp4a.40.2
mezzanine    webm       unknown    (best)
$
```
Until someone works out why the `mezzanine` format gives 403, the answer for both cases is to exclude the `webm` format, as above. 
```shellsession
$ python -m youtube_dl -v --test -f '[ext=mp4]' 'https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'--test', u'-f', u'[ext=mp4]', u'https://www.ign.com/videos/john-wick-4-teaser-trailer-comic-con-2022']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: 1ff18694c
[debug] Python version 2.7.17 (CPython) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[ign.com] john-wick-4-teaser-trailer-comic-con-2022: Downloading JSON metadata
[ign.com] 512a96f4-9e32-42b6-9b37-43b1e3e7eb1d: Downloading m3u8 information
[debug] Invoking downloader on u'https://assets.ign.com/videos/zencoder/2022/07/23/2560/512a96f4-9e32-42b6-9b37-43b1e3e7eb1d-1658545494.m3u8'
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 11
[download] Destination: John Wick 4 - Teaser Trailer - Comic-Con 2022-512a96f4-9e32-42b6-9b37-43b1e3e7eb1d.mp4
[download] 100% of 10.00KiB in 00:00
[debug] ffmpeg command line: ffprobe -show_streams 'file:John Wick 4 - Teaser Trailer - Comic-Con 2022-512a96f4-9e32-42b6-9b37-43b1e3e7eb1d.mp4'
[ffmpeg] Fixing malformed AAC bitstream in "John Wick 4 - Teaser Trailer - Comic-Con 2022-512a96f4-9e32-42b6-9b37-43b1e3e7eb1d.mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel 'repeat+info' -i 'file:John Wick 4 - Teaser Trailer - Comic-Con 2022-512a96f4-9e32-42b6-9b37-43b1e3e7eb1d.mp4' -c copy -f mp4 '-bsf:a' aac_adtstoasc 'file:John Wick 4 - Teaser Trailer - Comic-Con 2022-512a96f4-9e32-42b6-9b37-43b1e3e7eb1d.temp.mp4'
$
```
